### PR TITLE
Add DRAFT.3 for the feature isolation on tfvc article as a sample and…

### PR DESCRIPTION
**Pull request by wpschaub**
Share DRAFT.3 for **Effective feature isolation on TFVC** article to trigger review.

Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Guidance/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- DRAFT.3 of the feature isolation on tfvc article

@ALM-Rangers/guidance

### CODE REVIEW

File: src/Branching/effective-feature-isolation-on-tfvc.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+This topic aims to answer those questions, and give you a clear perspective on feature isolation in TFVC. (For guidance on TFVC branch strategies overall, read [Branching Strategies with TFVC](./branching-strategies-with-tfvc.md).) 
+
+> [!NOTE]
+> 
+> This topic covers TFVC. For Git, refer to [adopt a Git branching strategy](./git-branching-guidance.md). 
+
+#What's feature isolation?
+
+The feature isolation strategy allows you work on your features or fix bugs in **feature** branches (also known as “topic branches”) based off your **main** branch, or **dev**, or other central branches. It isolates your changes from the rest of your team, allows you to experiment with a feature, and enables easy rollback or merging of changes with your parent branch.
```
Comment (rhundhausen) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: Perhaps add a note differentiating the "Feature" work item type. Some readers may think that this pattern is only used when delivering things from that middle backlog level. Don't change the name of the strategy, just mention that it's used when developing PBIs or User Stories or whatever CMMI calls them. :-)

File: src/Branching/effective-feature-isolation-on-tfvc.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- Select **Tools > Options > Visual Studio Team Foundation Server**.
+- From the submenu, select **Show deleted items in the Source Control Explorer** to view the deleted branch(es). 
+
+If needed, you can also [restore](https://www.visualstudio.com/en-us/docs/tfvc/delete-restore-files-folders#restore-items-deleted-from-visual-studio) deleted branches and associated items. 
+
+![Show deleted feature branches](./_img/effective-feature-isolation-on-rfvc/feature-isolation-delete-branches-show.png)
+
+> [!NOTE]
+>
+> Use the [destroy](https://www.visualstudio.com/en-us/docs/tfvc/destroy-command-team-foundation-version-control) command with caution. It's a **permanent** delete!
```
Comment (rhundhausen) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: The Destroy note seems a little out of place as you didn't mention it up above.

File: src/Branching/effective-feature-isolation-on-tfvc.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+If no one in your team has destroyed a branch, you'll have the complete history needed for auditing and migration tools that rely on replaying history.
+
+So, by keeping your branches **short-lived** and adopting a **consistent naming convention**, the feature isolation strategy is effective for small and large teams.
```
Comment (rhundhausen) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: I would love it if you ended this guidance with something like ... as your team matures, consider moving to Continuous Integration, Feature Toggles, and trunk-based development. I'd be happy to write up a paragraph.

File: src/Branching/effective-feature-isolation-on-tfvc.md (hunk 1 above)
Comment (wpschaub) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: Good point. I'll add a note.

File: src/Branching/effective-feature-isolation-on-tfvc.md (hunk 2 above)
Comment (wpschaub) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: I'll add a paragraph before to talk about the soft  and hard (destroy) delete.

File: src/Branching/effective-feature-isolation-on-tfvc.md (hunk 3 above)
Comment (wpschaub) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: We're planning a feature toggle article.  Once available, we can add a closing link to the article.

Comment (wpschaub) [inline src/Branching/effective-feature-isolation-on-tfvc.md]: Then again we can add a note for user to consider feature toggles ad an alternative or complementary strategy. @rhundhausen thanks for all your feedback!

**Pull request merged**